SYCL/ESIMD/tpm_basic.cpp

```diff
@@ -9,9 +8,12 @@
 // TODO enable on Windows and Level Zero
 // REQUIRES: linux && gpu && opencl
 // RUN: %clangxx-esimd -fsycl %s -o %t.out
-// RUN: %HOST_RUN_PLACEHOLDER %t.out
 // RUN: %ESIMD_RUN_PLACEHOLDER %t.out
 
+// This test is intended to use Thread Private Memory (TPM) to support
```

Comment:
I don't quite understand, sorry. What is being tested? BE support for TPM-allocated variables?
BTW, there is no 'thread' concept in SYCL or ESIMD - workitem or subgroup. In ESIMD workitem == CM's thread, but please use SYCL terms.

Reply:
TPM is CM term and, as far as I know, it is not explicitly accessible on SYCL level and is used in case when variable can not be allocated on registers. These tests are to check the correctness of implementation of TPM in CM, that is used as ESIMD vector backend. I'm not sure how to write description in only SYCL terms.

Reply:
This is ESIMD test, ESIMD obeys SYCL execution and memory model, so in general it might not always be possible to test CM BE using ESIMD. These tests are under SYCL/ESIMD, so they should use SYCL/ESIMD only.

Reply:
AFAICS, CM TMP == SYCL private memory. Private is per-workitem memory.